doc/usage.md

```diff
@@ -39,31 +39,22 @@ The permission `createvault.forothers` allows you to create vaults for other pla
 Commands
 --------
 
-### User commands ###
-
-    /money
-Display your account's current balance. Alias `/m`
-
-    /money pay <amount> <player>
-Pay an amount to a player. The transaction will only succeed if your account has at least the given amount plus any taxes that apply, and the receiving account has enough capacity for the amount.
-
-    /money withdraw <amount>
-Withdraw an amount from chest storage into inventory.
+Arguments between `<>` (ex `<amount>`) are mandatory while ones between `[]` (ex. `[type]`) are optionals.
 
-    /money deposit <amount>
-Deposit an amount from inventory into chest storage. 
+### User commands ###
 
+| Command           | Arguments           | Description                             | Aliases         |
```

Comment:
I think the separate columns for command and arguments make the command structure less clear

Reply:
Hmm, ok you're right, i'll combine them.